packages/expo-permissions/ios/EXPermissions.podspec

```diff
@@ -13,15 +13,6 @@ Pod::Spec.new do |s|
   s.platform        = :ios, '10.0'
   s.source          = { git: 'https://github.com/expo/expo.git' }
   s.source_files    = "EXPermissions/*.{h,m}",
-                      # ready to export into separate modules
-                      "EXPermissions/Requesters/AudioRecording/*.{h,m}",
-                      "EXPermissions/Requesters/Brightness/*.{h,m}",
-                      "EXPermissions/Requesters/Calendar/*.{h,m}",
-                      "EXPermissions/Requesters/Camera/*.{h,m}",
-                      "EXPermissions/Requesters/CameraRoll/*.{h,m}",
-                      "EXPermissions/Requesters/Contacts/*.{h,m}",
-                      "EXPermissions/Requesters/Location/*.{h,m}",
-                      "EXPermissions/Requesters/Reminders/*.{h,m}",
```

Comment:
Why do we explicitly list all those files here? Couldn't we use a more general glob?

Reply:
It's here temporarily. I'll use general glob later.